CHANGES.rst

```diff
@@ -15,6 +15,7 @@ Improvements:
  * MXStore: Add a method to get related events for a specific event.
  * MXPublicRoom: Add canonical alias property.
  * MXLogger: Add a parameter to indicate the number of log files.
+ * Cross-Signing: Detect and expose new sign-ins (/vector-im/riot-ios/issues/2918).
```

Comment:
We should mention `MXDeviceListDidUpdateUsersDevicesNotification`.
All Cross-Signing things will finish in one line: *Add support of cross-signing*.